app/views/teachers/TeacherCourseSolutionView.coffee

```diff
@@ -33,20 +33,30 @@ module.exports = class TeacherCourseSolutionView extends RootView
       return '' if l isnt @language
       a
 
+  getText: (thang, what) ->
+    if thang.i18n
+      if i18n.lng() in Object.keys(thang.i18n)
+        return thang.i18n[i18n.lng()][what]
+    return thang[what]
+
```

Comment:
Should probably use our [utils.i18n function](https://github.com/codecombat/codecombat/blob/production/app/core/utils.coffee#L186-L210) like [this](https://github.com/codecombat/codecombat/blob/production/app/views/clans/ClanDetailsView.coffee#L188).